en-US/Samples/CompanionApp.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: sample
+title: Companion App (and Server)
+description: Create a simple app (for Windows Phone 10 or Android) that can configure your device's Wifi
+keyword: Windows 10 IoT Core, advanced, companion app, SoftAp, Wifi
+permalink: /en-US/Samples/CompanionApp.htm
+samplelink: N/A
+lang: en-US
+---
+
+# Design
+
+There are two parts to this sample:
+
+1. the Client - running on an Android or Windows Phone 10 phone, this app will find your IotCore device, connect to it, and allow you to specify the required information to join a WiFi network. 
+1. the Server - running on your IotCore device, this app (foreground or background) help the Client connect your IotCore device to a desired WiFi network. 
+
+### Load the project in Visual Studio
+
+You can find the source code for this sample by downloading a zip of all of our samples [here](https://github.com/ms-iot/samples/archive/develop.zip) and navigating to the `samples-develop\CompanionApp`.  Make a copy of the folder on your disk and open the project from Visual Studio.
```

Comment:
Generally want to avoid links using the word "here", maybe "though the Samples GitHub repository"?